src/main/java/jp/co/dwango/logback/SlackWebhookAppender.java

```diff
@@ -190,9 +180,22 @@ private void initScriptEngine() throws ScriptException {
         // define payload function
         StringBuilder function = new StringBuilder();
         function.append("function payload(event) {");
-        function.append(  "var color, emoji, level = event.getLevel().toString(),");
-        function.append(      "timestamp = formatTimestamp(event.getTimeStamp()),");
-        function.append(      "message = event.getFormattedMessage();");
+        function.append(  "var property = (function() {");
+        function.append(      "var lcvo = event.getLoggerContextVO(),");
+        function.append(          "properties = lcvo.getPropertyMap();");
+        function.append(      "return function(name) {");
+        function.append(        "if(!name && name !== 0) {");
+        function.append(          "return 'Property_HAS_NO_KEY';");
+        function.append(        "}");
+        function.append(        "name = '' + name;");
+        function.append(        "var value = properties.get(name);");
+        function.append(        "return value != null ? value : java.lang.System.getProperty(name);");
+        function.append(      "};");
```

Comment:
The property function implementation is based on [PropertyConverter](https://github.com/qos-ch/logback/blob/master/logback-classic/src/main/java/ch/qos/logback/classic/pattern/PropertyConverter.java) of logback. This is equivalent to [property conversion](https://logback.qos.ch/manual/layouts.html#property) of PatternLayout.